sycl/test/abi/layout_accessors_device.cpp

```diff
@@ -29,6 +29,7 @@ SYCL_EXTERNAL void hostAcc(accessor<int, 1, access::mode::read, access::target::
 // CHECK-NEXT: 16 |         size_t [1] common_array
 // CHECK-NEXT: 24 |   union sycl::accessor<int, 1, sycl::access::mode::read, sycl::access::target::global_buffer, sycl::access::placeholder::false_t>
 // CHECK-NEXT: 24 |     sycl::accessor<int, 1, sycl::access::mode::read, sycl::access::target::global_buffer, sycl::access::placeholder::false_t>::ConcreteASPtrType MData
+// CHECK-NEXT: 24 |     sycl::accessor<int, 1, sycl::access::mode::read, sycl::access::target::global_buffer, sycl::access::placeholder::false_t>::OCLImage1dBufferTy ImageBuffer 
```

Comment:
Please clarify if the ImageBuffer field was present before the change. Otherwise this would be ABI-breaking.

Reply:
Ok. I reverted this change and instead of breaking the ABI of SYCL device code, my patch now breaks ABI of ESIMD device code. Hope it is OK for us to do.